syncer.go

```diff
@@ -51,11 +51,11 @@ type Syncer struct {
 // NewSyncer instantiates the main stateful object in Keysync.
 func NewSyncer(config *Config, logger *logrus.Entry, metricsHandle *sqmetrics.SquareMetrics) (*Syncer, error) {
 	syncer := Syncer{config: config, clients: map[string]syncerEntry{}, logger: logger, metricsHandle: metricsHandle}
-	url, err := url.Parse("https://" + config.Server)
+	serverUrl, err := url.Parse("https://" + config.Server)
```

Comment:
This change doesn't affect anything, but gets rid of a warning about having a variable with the same name as an imported package.